plugins/guests/arch/cap/rsync.rb

```diff
@@ -0,0 +1,18 @@
+module VagrantPlugins
+  module GuestArch
+    module Cap
+      class RSync
+        def self.rsync_install(machine)
+          comm = machine.communicate
+          if !comm.test("test -f /usr/bin/rsync")
```

Comment:
Hi there,

This isn't the recommended way to do this anymore. We now have a capability for `rsync_installed` and a cap for `rsync_install`. The "installed" cap should perform the check if rsync is installed, and the install should actually install it.